packages/ui/src/components/no-search-results.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+
+import { NoData, NoDataProps } from './no-data'
+
+interface NoSearchResultsProps extends Omit<NoDataProps, 'iconSize'> {
+  iconSize?: number
+}
+
+export const NoSearchResults: React.FC<NoSearchResultsProps> = ({
```

Comment:
hm. dont we already have a no search results component? i see it being rendered in a few places

Reply:
So what already exists is the `no-data` component that is being used as a `no-search` component. The `no-search` component also internally uses the `no-data` component. Have removed for now, we can add back later if reqd.